soc/xtensa/intel_adsp/common/include/adsp-clk.h

```diff
@@ -24,37 +25,40 @@ void cavs_clock_init(void);
  *
  * @return 0 on success, -EINVAL if freq_idx is not valid
  */
-int cavs_clock_set_freq(uint32_t freq_idx);
+int adsp_clock_set_freq(uint32_t freq_idx);
 
 /** @brief Get list of cAVS clock information
  *
  * Returns an array of clock information, one for each core.
  *
  * @return array with clock information
  */
-struct cavs_clock_info *cavs_clocks_get(void);
+struct adsp_clock_info *adsp_clocks_get(void);
 
 /* Device tree defined constants */
-#define CAVS_SHIM_BASE          DT_REG_ADDR(DT_NODELABEL(shim))
-#define CAVS_SHIM_CLKCTL        (*((volatile uint32_t *)(CAVS_SHIM_BASE + 0x78)))
```

Comment:
@abonislawski : (https://github.com/zephyrproject-rtos/zephyr/pull/51919->Overall looks good, just please put hardcoded in code value of shim CLK register address to devicetree.) 
Done, please re-review :)